docker-compose.yml

```diff
@@ -44,9 +35,9 @@ services:
       interval: 5s
       timeout: 5s
       retries: 5
-    command: redis-server
+    command: redis-server --requirepass password
     ports:
-      - '6379:6379'
+      - '6380:6379'
```

Comment:
what's this change about?

Reply:
this is to prevent conflicts on a local computer that may already have PG or Redis running